packaging/ubports/manifest.json

```diff
@@ -9,7 +9,7 @@
             "desktop":  "pure-maps.desktop"
         }
     },
-    "version": "1.24.0",
+    "version": "1.24.1",
```

Comment:
let's put 1.24.0.1 here. I am sometimes using third number to update translations or fast bug fixes

Reply:
I have already released it like that. :see_no_evil:

Reply:
Next time I'll append a forth number if I do a Ubuntu Touch only patch release...

Reply:
OK, no problem. let's just keep it in mind. although, I hope we will not need any special UT releases :)